packages/rich-text-from-markdown/src/__test__/real-world.test.ts

```diff
@@ -158,6 +158,159 @@ describe('rich-text-from-markdown', () => {
             text('link with title'),
           ),
         ),
+        // Tables
+        block(BLOCKS.HEADING_2, {}, text('Tables')),
+        block(
+          BLOCKS.TABLE,
+          {},
+          block(
+            BLOCKS.TABLE_ROW,
+            {},
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('Name'))),
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('Country'))),
+          ),
+          block(
+            BLOCKS.TABLE_ROW,
+            {},
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('Contentful'))),
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('Germany'))),
+          ),
+          block(
+            BLOCKS.TABLE_ROW,
+            {},
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('Nasa'))),
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('USA'))),
+          ),
+          block(
+            BLOCKS.TABLE_ROW,
+            {},
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('> Google'))),
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('USA'))),
+          ),
+          block(
+            BLOCKS.TABLE_ROW,
+            {},
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('* Audi'))),
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('Germany'))),
+          ),
+          block(
+            BLOCKS.TABLE_ROW,
+            {},
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('# Volkswagen'))),
+            block(BLOCKS.TABLE_CELL, {}, block(BLOCKS.PARAGRAPH, {}, text('Germany'))),
+          ),
+          block(
+            BLOCKS.TABLE_ROW,
+            {},
+            block(
+              BLOCKS.TABLE_CELL,
+              {},
+              block(BLOCKS.PARAGRAPH, {}, text('Tesla')),
+              block(BLOCKS.PARAGRAPH, {}, text('SpaceX')),
```

Comment:
Consider this is
```md
 | <p>Tesla<br/>SpaceX</p>                 | USA     |
```
and we support soft linebreaks in CF RT, this probably shouldn't return two paragraphs if possible.

Reply:
@cribbles any opinion on this one :point_up: ?

Reply:
We receive two `text` node types containing `Tesla` and `SpaceX` as content. Them we wrap each one of them manually inside a paragraph. It's what the parser return. If this is something we should really support I can also spare some time trying to figure out another solution.

Reply:
Hmm, maybe add a test for the same case outside of a table - I assume it will be the same and I think we can go with that for now then.

I also wonder what a normal linebreak
```
text
more text in the next line

new paragraph
```
would translate to currently in md-to-rt. Rendered in MD it's
<img src="https://user-images.githubusercontent.com/101926/140305200-dc79f994-e148-4995-8ddc-ae4633e7195b.png" width="250" />

Reply:
Will do.